frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py

```diff
@@ -121,18 +128,19 @@ def calculate(self, a: str, b: str, mode: str, show_distance: bool = False):
             assert mode == "fhe", "Only 'simulate' and 'fhe' mode are available"
             self._compute_in_fhe([(a, b)], show_distance=show_distance)
 
-    def calculate_list(self, l: list, mode: str):
+    def calculate_list(self, list_to_compute_on: list, mode: str):
```

Comment:
name could be better.
pairs_to_measure/calculate

Reply:
I have set pairs_to_compute_on